var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -148,6 +152,7 @@ def url_for_version(self, version):
         return url
 
     def setup_environment(self, spack_env, run_env):
+        spack_env.set('MAKEFLAGS', '-j{0}'.format(make_jobs))
```

Comment:
This brings down the build time from more than 16 minutes to a bit less than 12 minutes on our 48-core machine.

I have also pushed an updated PR that drops skipping the script component.